.github/pull_request_template.md

```diff
@@ -5,19 +5,32 @@
 
 fixes/see #{{issue-no}} (use _see_ if issue requires QA and should not be closed automatically)
 
-# Review:
+# Review
 
 * [ ] code is understandable
 * [ ] code is covered by specs
 * [ ] no performance issues found
 * [ ] no security issues found
 
-# QA (after review is done)
+# QA
+
+After review is done.
+
+If no QA is needed, add "No QA" label before requesting a review.
+
+<details>
+  <summary>No QA criteria</summary>
+  <ul>
+    <li>API endpoints</li>
```

Comment:
I think it has at least once happened that during QA of an endpoint PR, the QA person found something that lead to a change, so the value of those is > 0.

The QA 'style' there was not "checking if things look good" and more "can I break this?". Or should this already be covered during the review step by the reviewer asking questions and the dev being responsible to check them?

Reply:
changed it a bit. people need to decide for themselves.